### ISSUE TITLE
Error while executing command, address may only be an extended SECP256k1 public key

### ISSUE THREAD

**Issue by lishuailibertine** [trustwallet/wallet-core#767]
**Describe the bug**
coinType :harmony
fool: walletconsole

**To Reproduce**
docker:  addrPub 03d4337374037cc8f81ccca1db1f32ee6cf7ecf357f3b4c9e9080960c218bb67ca
Error while executing command, address may only be an extended SECP256k1 public key

**Comment by hewigovens** [comment]
it's not a bug, Harmony team suggests to use un-compressed public key

**Comment by lishuailibertine** [comment]
&gt; addrPub 04d4337374037cc8f81ccca1db1f32ee6cf7ecf357f3b4c9e9080960c218bb67ca8c401b4bef5f47884a42decf456563e30e4fef4c96e7ddd8cce336f2069d066b
 
Error while executing command, Invalid public key data




compressed public key:&nbsp;03d4337374037cc8f81ccca1db1f32ee6cf7ecf357f3b4c9e9080960c218bb67ca




the top is&nbsp;un-compressed public key; does it generate wrong?&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "hewig"<notifications@github.com&gt;; 
发送时间: 2019年12月10日(星期二) 晚上6:34
收件人: "trustwallet/wallet-core"<wallet-core@noreply.github.com&gt;; 
抄送: "孤单男孩"<783367132@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [trustwallet/wallet-core] Error while executing command, address may only be an extended SECP256k1 public key (#767)




it's not a bug, Harmony team suggests to use un-compressed public key

**Comment by catenocrypt** [comment]
Indeed, for harmony the command sequence newKey -> pubPri -> addrPub fails, as you indicated.
As a workaround I propose the sequence newKey -> addrPri, this works.
I will check into the reason.

```
> coin harmony
Set active coin to: harmony    Use 'coin' to change.  (name: 'harmony'  symbol: one  numericalid: 1023)
> newkey
Result:  1ac575905b487fee54e2272fc5f0986c27718171538e3922e03ef50d7bdcb21d
> addrPri #
Result:  one1vturw97txmry9yfr0lxveaveq3sw4awmplk5g9
```

**Comment by catenocrypt** [comment]
Fix PR created (#770 ).  Thanks for the bug report!
